StaticResourceSources/customDatatable/style.css

```diff
@@ -0,0 +1,32 @@
+/* 
```

Comment:
I like this is a shared CSS library, I just wish we didn't have to bury it in a static resource. Unfortunately, it looks like it's the only way to do it right now. https://salesforce.stackexchange.com/questions/291394/importing-a-common-css-in-lwc